doc/setting_up_vcpkg.md

```diff
@@ -2,6 +2,43 @@
 
 This document describes how to setup vcpkg to build applications using Microsoft Azure IoT device C SDK. It demonstrates building and running a C language SDK sample for Windows, Linux, and Mac.
 
+## C SDK vcpkg support
```

Comment:
Nit - and it's bigger & pre-existing than this change since we're not consistent in docs.  But we should consider at least in this change "C SDK" => "IoT C SDK" or "Azure IoT C SDK" ??  While "C SDK" means something internally to us, for most customers they'll use many SDK's.

Fine to resolve for later discussion.